src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -244,3 +240,83 @@ def test_fetch_capture_waveform(multi_instrument_session):
         assert fetched_site == 1
         assert len(fetched_waveform[fetched_site]) == num_samples
 
+
+def test_get_pin_results_pin_information_with_channel_list(multi_instrument_session):
+    create_pin_map(multi_instrument_session)
+
+    fully_qualified_channels = [instr[1] + '/0', instr[0] + '/1']
+    pin_info = multi_instrument_session.channels[fully_qualified_channels].get_pin_results_pin_information()
+
+    pins = [i.pin_name for i in pin_info]
+    sites = [i.site_number for i in pin_info]
+    channels = [i.channel_name for i in pin_info]
+
+    assert pins == ['PinA', 'PinB']
+    assert sites == [1, 0]
+    assert channels == fully_qualified_channels
+
+
+def test_get_pin_results_pin_information_with_pin_list(multi_instrument_session):
+    create_pin_map(multi_instrument_session)
+
+    pin_info = multi_instrument_session.pins['SysPins, site0/DutPins'].get_pin_results_pin_information()
```

Comment:
I think it's more "pythonic" to pass multiple sites as an array.
`multi_instrument_session.pins[['SysPins'], ['site0/DutPins'].get_pin_results_pin_information()`

Consider doing things in what we consider "best practice" since people may look at test for usage examples.